requirements.txt

```diff
@@ -21,7 +21,7 @@ git+https://github.com/cos-forks/celery@v4.1.1+cos0
 kombu==4.2.0
 amqp==2.2.2
 vine==1.1.4
-itsdangerous==0.24
+itsdangerous==1.1.0
```

Comment:
Should this be listed in `Changes`?

Reply:
Apparently not necesary anyomore so removed.